src/integration-test/java/apptest/InvoiceSenderIT.java

```diff
@@ -1,109 +1,216 @@
 package apptest;
 
-import static apptest.util.TestUtil.extractZipFile;
-import static org.assertj.core.api.Assertions.assertThat;
-import static org.springframework.http.HttpMethod.POST;
-import static org.springframework.http.HttpStatus.OK;
-
-import java.io.File;
-import java.io.IOException;
-import java.time.Duration;
+import jcifs.CIFSContext;
+import jcifs.smb.SmbFile;
 import org.apache.commons.io.FileUtils;
-import org.junit.jupiter.api.AfterEach;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
```

Comment:
Just FYI. Your import settings are not up to date. use [this](https://github.com/Sundsvallskommun/.github-private/blob/main/profile/development-guidelines-api-team/sundsvall_profile_idea.xml) or you will have headaches when the next dept44 version rolls out